exercises/practice/sublist/.meta/config.json

```diff
@@ -1,6 +1,56 @@
 {
   "blurb": "Write a function to determine if a list is a sublist of another list.",
   "authors": [],
```

Comment:
another empty one, added in https://github.com/exercism/elixir/commit/6f82339d811367ac110e739d041e8c3ae7ae7b2d